package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "Microverse ES6 project",
+  "private": true,
+  "main": "src/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
You are missing necessary webpack scripts here as can be seen as a requirement from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)

Reply:
You should have build and dev commands for webpack